src/reducers/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-extraneous-dependencies */
+import { combineReducers } from 'redux';
+import books from './books';
+
+const newID = () => parseInt(Math.random() * 100, 10);
+export const initialState = {
```

Comment:
Just one thing to do as I mentioned in the previous review.
let's move `initialState` to `src/index.js`
![image](https://user-images.githubusercontent.com/31897434/76160627-fd564980-612b-11ea-82ab-7502490bb594.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/